x-pack/test/api_integration/deployment_agnostic/apis/observability/dataset_quality/check_and_load_integration.ts

```diff
@@ -0,0 +1,172 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { log, timerange } from '@kbn/apm-synthtrace-client';
+import expect from '@kbn/expect';
+
+import { DeploymentAgnosticFtrProviderContext } from '../../../ftr_provider_context';
+import { RoleCredentials, SupertestWithRoleScopeType } from '../../../services';
+
+export default function ({ getService }: DeploymentAgnosticFtrProviderContext) {
+  const samlAuth = getService('samlAuth');
+  const roleScopedSupertest = getService('roleScopedSupertest');
+  const synthtrace = getService('logsSynthtraceEsClient');
+  const packageApi = getService('packageApi');
+  const start = '2024-11-04T11:00:00.000Z';
+  const end = '2024-11-04T11:01:00.000Z';
+  const type = 'logs';
+  const dataset = 'synth';
+  const nginxDataset = 'nginx.access';
+  const apmDataset = 'apm.app.test';
+  const namespace = 'default';
+  const serviceName = 'my-service';
+  const hostName = 'synth-host';
+  const dataStreamName = `${type}-${dataset}-${namespace}`;
+  const nginxDataStreamName = `${type}-${nginxDataset}-${namespace}`;
+  const apmAppDataStreamName = `${type}-${apmDataset}-${namespace}`;
+
+  const pkg = 'nginx';
+
+  async function callApiAs({
+    roleScopedSupertestWithCookieCredentials,
+    apiParams: { dataStream },
+  }: {
+    roleScopedSupertestWithCookieCredentials: SupertestWithRoleScopeType;
+    apiParams: {
+      dataStream: string;
+    };
+  }) {
+    return roleScopedSupertestWithCookieCredentials.get(
+      `/internal/dataset_quality/data_streams/${dataStream}/integration/check`
+    );
+  }
+
+  describe('Check and load integrations', function () {
+    let adminRoleAuthc: RoleCredentials;
+    let supertestAdminWithCookieCredentials: SupertestWithRoleScopeType;
+
+    before(async () => {
+      adminRoleAuthc = await samlAuth.createM2mApiKeyWithRoleScope('admin');
+      supertestAdminWithCookieCredentials = await roleScopedSupertest.getSupertestWithRoleScope(
+        'admin',
+        {
+          useCookieHeader: true,
+          withInternalHeaders: true,
+        }
+      );
+      // Install nginx package
+      await packageApi.installPackage({
+        roleAuthc: adminRoleAuthc,
+        pkg,
+      });
+
+      await synthtrace.index([
+        // Ingest degraded data in Nginx data stream
+        timerange(start, end)
+          .interval('1m')
+          .rate(1)
+          .generator((timestamp) =>
+            log
+              .create()
+              .message('This is a log message')
```

Comment:
Is this really the message content, or was this stubbed and needs to be changed?

Reply:
Not sure if i understand the question, with synthtrace we generate Logs data and this is simply a generated log message

Reply:
ah ok, cool beans.